### ISSUE TITLE
Adding more metrics such as cache hit/miss

### ISSUE THREAD

**Issue by shouichi** [yabeda-rb/yabeda-rails#15]
Hi,

ActiveSupport publishes bunch of events and we want to instrument cache hit/miss through `cache_read.active_support`.

https://guides.rubyonrails.org/active_support_instrumentation.html

General question: we ultimately can instrument all events published by active support but maybe it's too much. Where should we draw a line?

Thank you.

**Comment by Envek** [comment]
I don't think that it would be useful to subscribe to all possible things from ActiveSupport.

But for things that would be useful to many users pull requests are definitely welcome!
